validator/src/main/resources/expected-data-template/otelSDKexpectedAWSSDKTrace.mustache

```diff
@@ -21,8 +21,12 @@
           "status": 200
         }
       },
+      "aws": {
+        "operation": "ListBuckets",
+      }
       "metadata": {
         "default": {
+          "aws.service": "s3",
```

Comment:
Not sure if this one is needed? Maybe AWS X-Ray backend uses this to set the `AWS::S3` origin?